website_sale_suggest_create_account/views/website_sale.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    
+    <template id="cart" inherit_id="website_sale.cart">
+        <!-- Show normal "Checkout" button if user is logged in or external login
+             is disabled -->
+        <xpath expr="//a[@href='/shop/checkout']" position="attributes">
+            <attribute name="t-if">(
+                user_id != website.user_id or
+                env['ir.config_parameter'].get_param(
+                    'auth_signup.allow_uninvited') != 'True' and
+                not optional_products and website_sale_order and
+                website_sale_order.website_order_line
+            )</attribute>
+        </xpath>
+    
+        <!-- Show choice in other cases -->
+        <xpath expr="//a[@href='/shop/checkout']" position="after">
+            <a t-if="user_id == website.user_id"
+               class="btn btn-primary pull-right mb32"
+               href="/web/login?redirect=/shop/checkout">
+                <span>Log in and checkout</span>
+                <span class="fa fa-long-arrow-right"/>
+            </a>
+            <t t-if="(user_id == website.user_id and
+                      env['ir.config_parameter'].get_param(
+                          'auth_signup.allow_uninvited') == 'True' and
+                      not optional_products and website_sale_order and
+                      website_sale_order.website_order_line)">
+                <a class="btn btn-primary pull-right mb32"
+                   href="/web/signup?redirect=/shop/checkout">
```

Comment:
Clicking on this gives a 404 error if I create the config parameter `auth_signup.allow_uninvited` and set it to `True`:

![captura de pantalla de 2016-11-03 11-15-22](https://cloud.githubusercontent.com/assets/973709/19962225/f515b608-a1b6-11e6-8c0d-8480304100bb.png)

I'm thinking that, given the controller is now for users, probably all these patches in the view are simply not required. Just go to `/shop/checkout` and you will get redirected to `/web/login?redirect=/shop/checkout` automatically, and enable the sign up option and there it will be. It is a good moment to drop many (if not all) of this XML if possible.